packages/runner/src/lib/SecScan.ts

```diff
@@ -83,6 +87,12 @@ export class SecScan {
     return this;
   }
 
+  public waitForCompletion(): SecScan {
```

Comment:
I would prefer a more idiomatic name
```suggestion
  public setFailFast(enable: boolean): SecScan {
```